rules/endpoints_test.go

```diff
@@ -288,10 +226,10 @@ var _ = Describe("Endpoints", func() {
 
 					{Action: ClearMarkAction{Mark: 0x8}},
 
-					{Comment: "Start of tier tier2",
+					{Comment: "Start of policies",
 						Action: ClearMarkAction{Mark: 0x10}},
 					{Match: Match().MarkClear(0x10),
-						Action: JumpAction{Target: "cali-po-tier2/c"}},
+						Action: JumpAction{Target: "cali-po-/c"}},
```

Comment:
The leading slash distinguishes between policies and profiles with the same name.  Was just trying to keep as much room for the name.